src/rules/no-done-callback.ts

```diff
@@ -41,17 +52,21 @@ export default createRule({
   create(context) {
     return {
       CallExpression(node) {
-        const callback = findCallbackArg(node);
+        // done is the second argument for it.each, not the first
+        const isJestEach = !!getNodeName(node.callee)?.endsWith('.each');
```

Comment:
```suggestion
        const isJestEach = getNodeName(node.callee)?.endsWith('.each') != null;
```

Reply:
Don't we have `eqeqeq` on? 🤔

Reply:
`.endsWith('.each')` returns `boolean`, I'm not sure if this has the same effect

Reply:
```suggestion
        const isJestEach = getNodeName(node.callee)?.endsWith('.each') ?? false;
```

Reply:
> Don't we have `eqeqeq` on? 🤔

I assume we allow "smart" equality which has exceptions for `== null`. regardless, that was the wrong thing 😀